firebase-devservices/deployment/src/test/java/io/quarkiverse/googlecloudservices/firebase/deployment/testcontainers/FirebaseEmulatorContainerCustomConfigTest.java

```diff
@@ -0,0 +1,88 @@
+package io.quarkiverse.googlecloudservices.firebase.deployment.testcontainers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.file.Files;
+
+import org.junit.jupiter.api.Test;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Testcontainers
+public class FirebaseEmulatorContainerCustomConfigTest {
+
+    private static final File tempEmulatorDataDir;
+
+    static {
+        try {
+            // Create a temporary directory for emulator data
+            tempEmulatorDataDir = Files.createTempDirectory("firebase-emulator-data").toFile();
```

Comment:
Better to create it somewhere it will be automatically deleted like in the target folder or /tmp

Reply:
true, on it

Reply:
Fixed https://github.com/quarkiverse/quarkus-google-cloud-services/pull/707/commits/338451a8c93df85c097d2ad005e9ccdf653c0a3a